products/google-nexus.md

```diff
@@ -0,0 +1,117 @@
+title: Google Nexus
+category: device
+tags: google mobile-phone
+iconSlug: google
+permalink: /nexus
```

Comment:
fyi, this permalink is already used: [Nexus Repository OSS](https://endoflife.date/nexus)

Reply:
Oh, you are right. Should I change it to "google-nexus"?

Reply:
Yes, I think that's a good idea.